### ISSUE TITLE
slick-greeter leaves at-spi process running

### ISSUE THREAD

**Issue by leigh123linux** [linuxmint/slick-greeter#13]
slick-greeter leaves at-spi process running which blocks user session at-spi

```

** (cinnamon-session:1614): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (cinnamon-settings-daemon:1797): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (cinnamon-launcher:1890): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (cinnamon:1932): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (polkit-gnome-authentication-agent-1:1948): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (canberra-gtk-play:1950): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (abrt:1942): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (nm-applet:1944): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (nemo:1946): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused
Failed to play sound: Sound disabled

** (cinnamon-killer-daemon:1947): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (blueberry-tray.py:1955): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (blueberry-obex-agent.py:1945): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused

** (redshift-gtk:1949): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-OWmOPJb639: Connection refused
Cjs-Message: JS LOG: About to start Cinnamon
St-Message: cogl npot texture sizes SUPPORTED
```

It is a known issue with lighdm-gtk

https://bugs.launchpad.net/debian/+source/lightdm/+bug/1366534

slick-greeter needs to clean up its child processes on exit.

**Comment by clefebvre** [comment]
Can't reproduce in Mint 18.1.

I added debug info to show the PID in /var/log/lightdm/seat0-greeter.log.

That PID exits properly here though.
